### PR TITLE
feat(frontend): new MessageBox level

### PR DESCRIPTION
**Pull request by DenysKarmazynDFINITY**
# Motivation

Add new level to the MessageBox component.

Screenshot 2024-11-11 at 12 28 52

### CODE REVIEW

File: src/frontend/src/lib/components/ui/MessageBox.svelte [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 	class="mb-4 flex items-start gap-4 rounded-xl px-4 py-3 text-sm font-medium sm:text-base"
 	class:bg-brand-subtle-alt={level === 'info'}
 	class:bg-warning-subtle={level === 'light-warning'}
+	class:bg-pale-pink={level === 'error'}
 >
 	<div
 		class="min-w-5 py-0 sm:py-0.5"
 		class:text-brand-primary={level === 'info'}
 		class:text-warning={level === 'light-warning'}
+		class:text-rusty-red={level === 'error'}
```
Comment (AntonioVentilii) [inline src/frontend/src/lib/components/ui/MessageBox.svelte:17]: with the new mapping here is `text-error`, I think

File: src/frontend/src/lib/components/ui/MessageBox.svelte [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
 
-	export let level: 'info' | 'light-warning' = 'info';
+	export let level: 'info' | 'light-warning' | 'error' = 'info';
 </script>
 
 <div
 	class="mb-4 flex items-start gap-4 rounded-xl px-4 py-3 text-sm font-medium sm:text-base"
 	class:bg-brand-subtle-alt={level === 'info'}
 	class:bg-warning-subtle={level === 'light-warning'}
+	class:bg-pale-pink={level === 'error'}
```
Comment (AntonioVentilii) [inline src/frontend/src/lib/components/ui/MessageBox.svelte:11]: and here it should be `bg-error-subtle-alt` (double-check please)

File: src/frontend/src/lib/components/ui/MessageBox.svelte (hunk 1 above)
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/lib/components/ui/MessageBox.svelte]: Done.

File: src/frontend/src/lib/components/ui/MessageBox.svelte (hunk 2 above)
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/lib/components/ui/MessageBox.svelte]: Yes, thanks!

**Review by AntonioVentilii — APPROVED** [review]
LGTM! Tks

**Pull request merged**